lib/event_store/append_to_stream.ex

```diff
@@ -85,4 +97,18 @@ defmodule Essig.EventStore.AppendToStream do
       events -> {:ok, Enum.reverse(events)}
     end
   end
+
+  defp signal_new_events(stream_uuid) do
+    scope_uuid = Essig.Context.current_scope()
+    bin_uuid = Ecto.UUID.dump!(scope_uuid)
+    stream_uuid = Ecto.UUID.dump!(stream_uuid)
```

Comment:
I can also send max_id and count here, so that we dont need extra queries to figure out, if we got all the recent events from the events table